earth2studio/models/px/sfno.py

```diff
@@ -150,6 +151,7 @@ def __init__(
         lon = np.linspace(0, 360, 1440, endpoint=False)
         self.input_coords = OrderedDict(
             {
+                "batch": np.empty(0),
                 "time": np.empty(1),
```

Comment:
Is time a free dim? (like can be any length?) Should update to np.empty(0).

Reply:
I think we can treat it as a free dim, but it is required.